sdk/component/go.mod

```diff
@@ -0,0 +1,14 @@
+module github.com/ocurity/dracon/sdk/component
```

Comment:
❓  In my head it makes sense to have a `sdk/component` module but I also can see a world where we just have a `sdk` module, in case we want to add more functionalities to the SDK (for example for workflow management).

I still think that the first option makes more sense as it's more isolated. Let me know what you think.

Reply:
`sdk/component` is clearer IMHO.

Reply:
Opted for module at `sdk` level so we can use the same module for a future `workflow` package and similar.

Still would work nice with `component.Stuff` or `workflow.Stuff` with imports

`github.com/ocurity/dracon/sdk/component` and github.com/ocurity/dracon/sdk/workflow